doc: update documentation (#676)
MovingBlocks/TerasologyLauncher: docs/CONTRIBUTING.md

# Contributing to TerasologyLauncher

If you would like to contribute code, documentation, or other assets you can do so through GitHub by forking the repository and sending a pull request (PR).
You can also simply report issues (bugs), but please search for any previous reports first.

_When submitting code, please make every effort to follow existing conventions and style in order to keep the code as readable as possible._

Read on for an overview.
For questions please join us in our [forum] or [Discord].

## File an Issue

You can report bugs and feature requests to [GitHub Issues](https://github.com/MovingBlocks/TerasologyLauncher/issues). As mentioned please look for a similar existing issue before submitting a new one.

> For finding easy to do issues to start with look at the [Good First Issue](https://github.com/MovingBlocks/TerasologyLauncher/labels/Good%20First%20Issue) issues.

Reviewer: ```suggestion
For finding easy to do issues to start with, look at the [Good First Issue](https://github.com/MovingBlocks/TerasologyLauncher/labels/Good%20First%20Issue) issues.
```

> We prefer questions and support requests be posted in the [forum] or [Discord].

Reviewer: ```suggestion
We prefer questions and support requests to be posted on [Discord].
```
